Doc/library/sqlite3.rst

```diff
@@ -1345,27 +1347,23 @@ The following example illustrates the implicit and explicit approaches:
 .. literalinclude:: ../includes/sqlite3/converter_point.py
 
 
-Default adapters and converters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There are default adapters for the date and datetime types in the datetime
-module. They will be sent as ISO dates/ISO timestamps to SQLite.
+Default adapters and converters (deprecated)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The default converters are registered under the name "date" for
-:class:`datetime.date` and under the name "timestamp" for
-:class:`datetime.datetime`.
-
-This way, you can use date/timestamps from Python without any additional
-fiddling in most cases. The format of the adapters is also compatible with the
-experimental SQLite date/time functions.
+.. note::
```

Comment:
I'm not sure. My first reaction is that the warning directive is a little bit over the top here. I'm leaning against keeping `note`.

Reply:
The standard way for the stdlib reference is using the ``deprecated-removed` directive, which is also used here below. I'm not sure if there's any real standard for "how-to guide" material like this, though personally I'd lean toward drawing more attention to features being deprecated rather than less, and risking people glancing over it. However, it ultimately appears to be at the discretion of the doc authors and module maintainers, i.e. @erlend-aasland .

Reply:
As there is no precedence for using the `warning` directive for these things, I'll keep it as a `note`. Note that `note` is used in a handful of places in `Doc/library`, where deprecation is mentioned in some kind of way.